### DISCUSSION TITLE
Change DefaultCartesianMarker value to include String

### DISCUSSION THREAD

**Discussion by Maartinsh** [patrykandpatrick/vico#689] [Questions] [answered]
Currently I have modified `DefaultCartesianMarker` to my needs and get a Int value, which is great, for instance: `8`.
I'd like to print out something like `8 units` where units is custom String.

I'm using Views approach:

```
DefaultCartesianMarker(
        label = TextComponent.build {
            textAlignment = Layout.Alignment.ALIGN_CENTER
            textSizeSp = 14f
        },
        labelPosition = DefaultCartesianMarker.LabelPosition.AroundPoint,
        valueFormatter = DefaultCartesianMarkerValueFormatter(colorCode = false),
    )
```

**Comment by Maartinsh** [comment]
Still seeking for an answer, thanks.

**Comment by patrickmichalik** [comment, marked as answer]
Hello! `DefaultCartesianMarkerValueFormatter` has a `decimalFormat` constructor parameter—you can apply a `DecimalFormat` instance that adds a suffix. To add a “units” suffix, you could use something like this:

```kt
DefaultCartesianMarkerValueFormatter(decimalFormat = DecimalFormat("#.## units"), colorCode = false)
```
